cli/command/utils.go

```diff
@@ -16,3 +18,24 @@ func extractArgs(received []string, names ...string) (map[string]string, error)
 
 	return args, nil
 }
+
+func resolveSingleEntityID(resolver resolver.Resolver, entityType, target string) (string, error) {
```

Comment:
we can remove this - the actual implementation is in `base.go` and already merged.

Reply:
✅